apps/src/javalab/Javalab.js

```diff
@@ -192,6 +193,7 @@ Javalab.prototype.init = function (config) {
     isResponsive: true,
     isSubmittable: !!config.level.submittable,
     isSubmitted: !!config.level.submitted,
+    recaptchaSiteKey: config.level.recaptchaSiteKey,
```

Comment:
Do we have any concerns about someone grabbing this key and spamming the captcha endpoint?